client/templates/fulfillmentOrders/fulfillmentOrders.html

```diff
@@ -37,7 +36,7 @@
             <th><i class='fa fa-truck'></i></th>
             <th>Ship By</th>
             <th class="hidden-xs hidden-sm">Arrival Day</th>
-            <th class="hidden-xs hidden-sm">First Ski Day</th>
+            <th class="hidden-xs hidden-sm">First Use Day</th>
```

Comment:
I'm thinking we should change this to "Reservation Start" or something similar as we expand beyond our original Ski / Camping product focus.